### PR TITLE
Times Lodash updates

### PR DESCRIPTION
**Pull request by GeorgeWL**

### CODE REVIEW

**Comment by GeorgeWL** [comment]
I have just realised that this isn't the right method of pull right? need to update the all.js in /tests ?

**Comment by cjtaylor1990** [comment]
> I have just realised that this isn't the right method of pull right? need to update the all.js in /tests ?

I would certainly say that you should add in a test in ./tests/unit/all.js. This pretty much just checks to make sure that the native result is equal to the result from the lodash method.

**Comment by cht8687** [comment]
@GeorgeWL Thanks for the contribution. Yes, agree with @cjtaylor1990, would be great to update the test cases in here: https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore/blob/master/tests/unit/all.js#L101

File: README.md [hunk 1]
```diff
@@ -2526,6 +2526,43 @@ Invokes the iteratee n times, returning an array of the results of each invocati
   var result = [...Array(10).keys()]
   console.log(result)
   // output: '[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]'
+  
+  // Lodash
+  var result = _.times(10,'a')
+  console.log(result)
+  // output  ['a', 'a', 'a', 'a', 'a', 'a', 'a', 'a', 'a', 'a' ]
```
Comment (GeorgeWL) [inline README.md]: didn't double check this one. Lodash returns an array of 10 undefined

**Comment by GeorgeWL** [comment]
not needed. I thought this behaviour wasn't existing, but I simply wasn't using the same syntax that lodash expected.

**Pull request closed without merging**